src/main/java/de/qabel/desktop/ui/about/AboutController.java

```diff
@@ -1,131 +1,118 @@
 package de.qabel.desktop.ui.about;
 
+import de.qabel.desktop.config.FilesAbout;
 import de.qabel.desktop.ui.AbstractController;
 import de.qabel.desktop.ui.about.aboutPopup.AboutPopupController;
 import de.qabel.desktop.ui.about.aboutPopup.AboutPopupView;
 import javafx.fxml.FXML;
 import javafx.fxml.Initializable;
 
-import javafx.scene.*;
-import javafx.scene.control.*;
 import javafx.scene.control.Label;
 import javafx.scene.layout.*;
-import org.w3c.dom.Document;
-import org.w3c.dom.NodeList;
-import org.xml.sax.SAXException;
 
 import javax.inject.Inject;
-import javax.xml.parsers.DocumentBuilderFactory;
-import javax.xml.parsers.ParserConfigurationException;
-import java.awt.Desktop;
-import java.io.*;
-import java.net.URI;
 import java.net.URL;
 import java.util.*;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
 
 public class AboutController extends AbstractController implements Initializable {
-    private static final String LICENSE_DEPENDENCY_MAPPING_LOCATION = "/license-dependency.xml";
-    private static final String DEPENDENCY_LICENSE_MAPPING_LOCATION = "/dependency-license.xml";
-    private static final ExecutorService executor = Executors.newCachedThreadPool();
 
     @FXML
-    Pane linkContainer;
+    private Pane linkContainer;
 
     @FXML
-    private Button thanksButton;
+    private Label appVersion;
+
+    @FXML
+    private Pane QAPLBox;
+
+    @FXML
+    private Pane thanksBox;
+
+    @FXML
+    private Pane termsOfServiceBox;
+
+    @FXML
+    private Pane imprintBox;
+
+    @FXML
+    private Pane privacyNotesBox;
+
+    @FXML
+    private Pane apacheLicenseBox;
+
+    @FXML
+    private Pane silLicenseBox;
+
+    @FXML
+    private Pane attributionLicenseBox;
+
+    @FXML
+    private Pane lgplLicenseBox;
+
+    @FXML
+    private Pane creativeLicenseBox;
 
     @Inject
     private Pane layoutWindow;
 
     @Inject
-    private String thanksFileContent;
+    private String currentVersion;
+
+    @Inject
+    private FilesAbout aboutFilesContent;
 
     public AboutPopupController popupController;
     private AboutPopupView popupView;
 
-    private Map<String, String> jarNames = new HashMap<>();
     private ResourceBundle resourceBundle;
 
     @Override
     public void initialize(URL location, ResourceBundle resources) {
         resourceBundle = resources;
         try {
-            addDependencyLicenses();
-            activateLinks();
             activateButtons();
             initializePopup();
+            setDesktopVersion();
         } catch (Exception e) {
             alert("failed to load about contents: " + e.getMessage(), e);
         }
     }
 
-    private void addDependencyLicenses() throws ParserConfigurationException, IOException, SAXException {
-        indexJarNames();
-        addLicenses();
```

Comment:
We can't just drop that code.
It looks like the mockups now but they are just static examples.
This code loaded the `license-dependency.xml` and `dependency-license.xml` which are both generated at distribution time from the real gradle dependencies. Thus, it is (as much as we can) guaranteed that they include all dependencies we actually have automatically. And we need to list all of them.

Not only are about 90% of them missing now but also many licenses we use like `source sans pro` and the `Bouncy Castle License` are missing. I'm going to add a screenshot of the current release to show an example what it looked like with the real content. We need to list all deps, and all licenses.